### PR TITLE
#92 - Updated pom to force maven-updater-plugin to version 2.0.0

### PR DESCRIPTION
**Pull request by PeteKeller**
@yegor256 This is a temporary fix to get the build working.  As I work through the dependencies I will be working this update into the parent or base pom.xml.

### CODE REVIEW

**Comment by dmarkov** [comment]
@petekeller Thanks, someone will review your pull request soon

File: pom.xml [hunk 1]
```diff
@@ -116,6 +116,28 @@
                 <extensions>true</extensions>
             </plugin>
         </plugins>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <artifactId>maven-invoker-plugin</artifactId>
+                    <version>2.0.0</version>
```
Comment (yegor256) [inline pom.xml]: @PeteKeller you don't need to repeat the entire configuration from the parent POM here, just provide the version and everything else will be inherited

**Comment by yegor256** [comment]
@PeteKeller looks good, but there is a comment from me above

**Comment by PeteKeller** [comment]
@yegor256 I updated my merge to remove the configuration.

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by dmarkov** [comment]
@rultor deploy

**Pull request merged**